standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/utils/MetaStoreUtils.java

```diff
@@ -291,9 +291,17 @@ public static boolean isTargetOfReplication(Database db) {
     return dbParameters != null && !StringUtils.isEmpty(dbParameters.get(ReplConst.TARGET_OF_REPLICATION));
   }
 
+  public static boolean forceEnableBackgroundThreads(Database db) {
```

Comment:
is this `isBackgroundTreadsEnabled`?
The current name suggests for me that we set something forcefully, but the code shows that we are just checking a property